app/views/employees/edit.html.slim

```diff
@@ -0,0 +1,10 @@
+.container
+	h1 Editing employee
+
+	.float-left
+		== render 'form'
+
+	= button_to 'Show', @employee, :method => :get
```

Comment:
want to use the button method to create buttons for users to click on, does this need to be changed